package/boot/uboot-mvebu/patches/0002-arm-mvebu-mvebu_armada-37xx-Define-the-loadaddr-envi.patch

```diff
@@ -0,0 +1,35 @@
+From 7977bb60120bfe274829829e5130261322ce8b2d Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Kerma=20G=C3=A9rald?= <gandalf@gk2.net>
+Date: Wed, 11 Aug 2021 08:35:40 +0200
+Subject: [PATCH 2/3] arm: mvebu: mvebu_armada-37xx: Define the loadaddr
```

Comment:
Same as above. Keep the original author.

Reply:
From is for sender…
Author was in cc